### DISCUSSION TITLE
Demand & Purchase Order Generation

### DISCUSSION THREAD

**Discussion by neokleon** [snipe/snipe-it#14756] [General]
Hi,

Can i ask how do i put a new module for creating Demands note and Purchase order 
Demand Module
1. Demand note have unique serial number
2. for creating demand of an asset , a new asset model must be created
3. quantities can be called
4. user with department can be called
5. Demand date is added (Current)
6. location can be called
7. Remarks can be added
|Report can be printed|
|add demands link in Dash board|
Demand Note must be approved for calling with Purchase Order (Approved Button)

Purchase Order Module
1. Purchase Order note have unique serial number
2. for creating Purchase Order , Demand can be called from demand number
3. quantities can be called from demand note
4. user with department can be called from demand note
5. Purchase Order date should be added (Current)
6. location can be called from demand note
7. Remarks can be added
|Report can be printed|
|add Purchase Orders link in Dash board|
PO Note must be approved for calling assets in (Approved Button)

Gate Pass Module
1. Purchased items can be called from PO for gate pass
2. GP# auto generated
3. GP items called from PO
4. received quantities can be added
5. called items quantity will be subtracted from received qtys and balance made.

Inspection Module
1. GP received items are called in inspection form
2. Auto serial for Inspection form
3. here received items are shown to put their details i.e (Brand, Model, Serial, MAC LAN, MAC WIFI, HDD, RAM, etc) |FieldSet for certain asset type|
4. Inspected items are added to assets for checkout

All that i want 

Regards

**Comment by rhenustr** [comment]
is such a thing possible

in addition, a department manager will approve it, and then, according to the approval, the director will approve it and send an email to the accounting department
@snipe
